receiver/hostmetricsreceiver/factory.go

```diff
@@ -94,40 +92,14 @@ func createMetricsReceiver(
 		return nil, err
 	}
 
-	schemaURLSetterConsumer, err := wrapBySchemaURLSetterConsumer(consumer)
-	if err != nil {
-		return nil, err
-	}
```

Comment:
We do lose `SchemaURL` on the metrics that don't have any semantic conventions attributes.

Reply:
I may have misunderstood the effect of this change.

So, previously we were setting the SchemaURL for all scrapers. After this change we will only set it for process scraper, for all other scrapers the SchemaURL will remain empty. This is because those other scrapers don't really confirm with Otel semantic conventions. Is this correct? I would expect that cpu scraper for example confirms with Otel conventions. Is it incorrectly implemented?

Reply:
> After this change we will only set it for process scraper, for all other scrapers the SchemaURL will remain empty. 

Correct

> This is because those other scrapers don't really confirm with Otel semantic conventions. Is this correct? I would expect that cpu scraper for example confirms with Otel conventions. Is it incorrectly implemented?

Hm, it does actually comply with the Otel semantic conventions in terms of metrics and attribute names, it just doesn't use resource attributes definitions generated from the semantic conventions. I think misinterpreted the meaning of the `SchemaURL` in that case. I'll return the `SchemaURL` back to all the host metrics. And once we have generated semconv definitions for metrics we can utilize them the same way as we use resource attributes definitions.

Reply:
Yes, the SchemaURL in the ResourceMetrics applies not just to Resource attributes but also to Metric attributes/values. So, if a scraper uses Otel-compliant metric attribute names then the SchemaURL is applicable to it too.